package.json

```diff
@@ -39,12 +39,13 @@
     "chalk": "^2.4.1",
     "commander": "^2.18.0",
     "ejs": "^2.6.1",
-    "express": "^4.16.3",
     "filesize": "^3.6.1",
     "gzip-size": "^5.0.0",
     "lodash": "^4.17.15",
     "mkdirp": "^0.5.1",
     "opener": "^1.5.1",
+    "polka": "^0.5.2",
+    "serve-static": "^1.14.1",
```

Comment:
I would suggest [`sirv`](https://github.com/lukeed/sirv) here instead for further size (and performance) improvements. Unfortunately `serve-static` is a significant portion of Express' size. You can find an example with Polka [here](https://github.com/lukeed/polka/blob/master/examples/with-sirv/index.js)